### PR TITLE
fix rpm spec isses

### PR DESCRIPTION
**Pull request by mwperina**
- Fix rpm-lint summary issues
- Fix rpm-lint description-line-too-long issue
- Fix rpm-lint conffile-without-noreplace-flag issue

### CODE REVIEW

**Comment by mwperina** [comment]
Fixed:
```
hirte.x86_64: E: summary-too-long Hirte is a service orchestrator tool intended for multi-node devices (e.g.: edge devices) clusters
hirte.x86_64: W: name-repeated-in-summary Hirte
hirte.x86_64: E: description-line-too-long Hirte is a service orchestrator tool intended for multi-node devices (e.g.: edge devices) clusters with a predefined
hirte.x86_64: E: description-line-too-long number of nodes and with a focus on highly regulated environment such as those requiring functional safety (for example
hirte-agent.x86_64: E: description-line-too-long Hirte is a service orchestrator tool intended for multi-node devices (e.g.: edge devices) clusters with a predefined
hirte-agent.x86_64: E: description-line-too-long number of nodes and with a focus on highly regulated environment such as those requiring functional safety (for example
hirte.x86_64: W: conffile-without-noreplace-flag /etc/hirte/hirte.conf
hirte-agent.x86_64: W: conffile-without-noreplace-flag /etc/hirte/agent.conf
```

Leave as is, because we want that behavior:
```
hirte.x86_64: W: non-conffile-in-etc /etc/dbus-1/system.d/org.containers.hirte.conf
hirte-agent.x86_64: W: non-conffile-in-etc /etc/dbus-1/system.d/org.containers.hirte.Agent.conf
```

Leave as is for now, it can be fixed later:
```
hirte.x86_64: W: dbus-policy-allow-receive <allow receive_sender="org.containers.hirte"/> /etc/dbus-1/system.d/org.containers.hirte.conf
hirte.x86_64: W: dbus-policy-allow-receive <allow receive_sender="org.containers.hirte"/> /etc/dbus-1/system.d/org.containers.hirte.conf
hirte-agent.x86_64: W: dbus-policy-allow-receive <allow receive_sender="org.containers.hirte.Agent"/> /etc/dbus-1/system.d/org.containers.hirte.Agent.conf
hirte-agent.x86_64: W: dbus-policy-allow-receive <allow receive_sender="org.containers.hirte.Agent"/> /etc/dbus-1/system.d/org.containers.hirte.Agent.conf
```

All errors has been fixed, so rpm-lint should pass now

File: hirte.spec.in [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Name:		hirte
 Version:	@VERSION@
 Release:	@RELEASE@%{?dist}
-Summary:	Hirte is a service orchestrator tool intended for multi-node devices (e.g.: edge devices) clusters
+Summary:	Service orchestrator tool for multi-node devices (e.g.: edge devices)
```
Comment (pypingou) [inline hirte.spec.in]: I would have started with `A`:
```suggestion
Summary:	A service orchestrator tool for multi-node devices (e.g.: edge devices)
```

**Review by engelmi — APPROVED** [review]
LGTM

File: hirte.spec.in (hunk 1 above)
Comment (mwperina) [reply hirte.spec.in]: Inspired in systemd, which has:

```
Summary:        System and Service Manager
```

But of course I can change if you think that with 'A' it's better

Comment (mwperina) [reply hirte.spec.in]: OK, 'A' added

**Review by pypingou — APPROVED** [review]

**Pull request merged**